### PR TITLE
Updating MSMQ tests to use ConsoleMF

### PR DESCRIPTION
**Pull request by chynesNR**
* Converted MSMQ tests to ConsoleMF
* Removed dedicated MSMQ test application (no longer needed)
* Tests now cover .NET Framework 4.6.2, 4.7.1, and 4.8 (.NET Core does not support MSMQ)

### CODE REVIEW

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs [hunk 1]
```diff
@@ -42,9 +42,6 @@ public StackExchangeRedisTestsBase(TFixture fixture, ITestOutputHelper output) :
                 exerciseApplication: () => _fixture.AgentLog.WaitForLogLines(AgentLogBase.TransactionTransformCompletedLogLineRegex, TimeSpan.FromMinutes(2), 2)
             );
 
-            // Confirm both transaction transforms have completed before moving on to host application shutdown, and final sendDataOnExit harvest
```
Comment (nr-ahemsath) [inline tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs:45]: Did you mean to remove this?

**Review by nr-ahemsath — CHANGES_REQUESTED** [review]
I had one comment about a line removed from an unrelated Redis test.  Otherwise it looks good.

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs (hunk 1 above)
Comment (chynesNR) [reply tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs:45]: Yeah, it was a merge error from my last PR. It's duplicated code.

Comment (nr-ahemsath) [reply tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs:45]: Got it, thanks!

**Review by nr-ahemsath — APPROVED** [review]
Approved.  Tests pass locally and in CI.

**Pull request merged**